### PR TITLE
PHP Warnings in Typo3 11 with PHP 8

### PR DESCRIPTION
**Pull request by SSFGizmo**
https://github.com/georgringer/news/issues/1630

### CODE REVIEW

File: Classes/Controller/AdministrationController.php [hunk 1]
```diff
@@ -389,16 +395,17 @@ public function indexAction(): void
...
+        $dblist->noControlPanels = isset($this->tsConfiguration['controlPanels']) ? !(bool)$this->tsConfiguration['controlPanels'] : true;
+        $colums = (isset($this->tsConfiguration['columns']) && !empty($this->tsConfiguration['columns'])) ? $this->tsConfiguration['columns'] : 'teaser,istopnews,datetime,categories';
         $dblist->setFields = [
-            'tx_news_domain_model_news' => GeneralUtility::trimExplode(',', $this->tsConfiguration['columns'] ?: 'teaser,istopnews,datetime,categories', true)
+            'tx_news_domain_model_news' => GeneralUtility::trimExplode(',', $colums, true)
         ];
 
         $tableRendering = $dblist->generateList();
-        if (!$tableRendering) {
+        //if (!$tableRendering) {
```
Comment (brotkrueml) [inline Classes/Controller/AdministrationController.php:405]: Why commented out?

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Classes/Backend/RecordList/NewsDatabaseRecordList.php [hunk 2]
```diff
@@ -34,9 +34,9 @@ public function listURL($alternativeId = '', $table = '-1', $excludeList = ''):
         } else {
             $urlParameters['id'] = $this->id;
         }
-        if ($this->thumbs) {
-            $urlParameters['imagemode'] = $this->thumbs;
-        }
+        //if ($this->thumbs) {
```
[the remark this answers is not in the document]
Comment (SSFGizmo) [reply Classes/Backend/RecordList/NewsDatabaseRecordList.php:37]: https://github.com/georgringer/news/issues/1630

File: Classes/Backend/RecordList/NewsDatabaseRecordList.php [hunk 3]
```diff
@@ -34,8 +34,15 @@ public function listURL($alternativeId = '', $table = '-1', $excludeList = ''):
         } else {
             $urlParameters['id'] = $this->id;
         }
-        if ($this->thumbs) {
-            $urlParameters['imagemode'] = $this->thumbs;
+        // todo can be remove when 10.4 supported dropped
+        $versionInformation = GeneralUtility::makeInstance(\TYPO3\CMS\Core\Information\Typo3Version::class);
```
Comment (brotkrueml) [inline Classes/Backend/RecordList/NewsDatabaseRecordList.php:38]: No need to use GU, just `(new \TYPO3\CMS\Core\Information\Typo3Version())->getMajorVersion()` as this isn't a service.

File: Classes/Controller/AdministrationController.php [hunk 4]
```diff
@@ -377,7 +383,11 @@ public function indexAction(): void
         $dblist->allFields = 1;
         $dblist->displayFields = false;
         $dblist->dontShowClipControlPanels = true;
-        $dblist->counter++;
+        //todo can be remove when 10.4 supported dropped
+        $versionInformation = GeneralUtility::makeInstance(\TYPO3\CMS\Core\Information\Typo3Version::class);
```
Comment (brotkrueml) [inline Classes/Controller/AdministrationController.php:387]: No need to use GU, just `(new \TYPO3\CMS\Core\Information\Typo3Version())->getMajorVersion()` as this isn't a service.

**Comment by georgringer** [comment]
this has been now all fixed, therefore closing this PR

**Pull request closed without merging**